### DISCUSSION TITLE
casbin-v

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#23325] [General]
https://github.com/casbin/casbin

Cashin implemented in V language

**Comment by Jengro777** [comment]
@medvednikov 
Does V currently have a project similar to cashbin? Building enterprise level projects requires similar permission management

**Reply by JalonSolov** [reply]
No, V does not currently have an equivalent module.  Others are asking for it, though.

**Reply by Jengro777** [reply]
If you want to use V to create an enterprise level application, this is necessary. If you expect someone to make it

**Reply by JalonSolov** [reply]
I'm sure someone will.  And no, it is not "necessary", as you could handle that in caddy/nginx proxy in front of your V code.

It would just make it simpler to deploy if you didn't need caddy/nginx as well.
